addons/payment_custom/views/payment_provider_views.xml

```diff
@@ -14,6 +24,19 @@
                     {'invisible': [('code', '=', 'custom')]}
                 </attribute>
             </group>
+            <field name="pending_msg" position="after">
+                <div class="o_row replace_pending_msg" colspan="2"
```

Comment:
What is this new class?  It doesn't seem to do anything to me :thinking:

Reply:
it does nothing indeed